src/praisonai-code/praisonai_code/cli/commands/rules.py

```
@@ -1,41 +1,130 @@
+        all_rules = manager.get_all_rules()
+    except Exception as exc:  # noqa: BLE001
+        typer.echo(f"Failed to load rules: {exc}", err=True)
+        raise typer.Exit(1)
+
+    removed = 0
+    for rule in all_rules:
+        try:
+            if manager.delete_rule(rule.name):
+                removed += 1
+        except Exception:  # noqa: BLE001 - skip undeletable (e.g. discovered) rules
+            continue
```

Ruff (0.16.2)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)